src/pages/groupComparison/CopyNumberEnrichments.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from "react";
+import {observer} from "mobx-react";
+import EnrichmentsDataSetDropdown from "../resultsView/enrichments/EnrichmentsDataSetDropdown";
+import AlterationEnrichmentContainer from "../resultsView/enrichments/AlterationEnrichmentsContainer";
+import GroupComparisonStore from "./GroupComparisonStore";
+import autobind from "autobind-decorator";
+import {MolecularProfile} from "../../shared/api/generated/CBioPortalAPI";
+import {MakeMobxView} from "../../shared/components/MobxView";
+import Loader from "../../shared/components/loadingIndicator/LoadingIndicator";
+import ErrorMessage from "../../shared/components/ErrorMessage";
+
+export interface ICopyNumberEnrichmentsProps {
+    store: GroupComparisonStore
+}
+
+@observer
+export default class CopyNumberEnrichments extends React.Component<ICopyNumberEnrichmentsProps, {}> {
+    @autobind
+    private onChangeProfile(m:MolecularProfile) {
+        this.props.store.copyNumberEnrichmentProfile = m;
+    }
+
+    readonly tabUI = MakeMobxView({
+        await:()=>[
+            this.props.store.copyNumberHomdelEnrichmentData,
+            this.props.store.copyNumberAmpEnrichmentData
+        ],
+        forcePending:()=>!this.props.store.copyNumberEnrichmentProfile,
```

Comment:
what is forcePending about?

Reply:
its like await, but with any predicate. forcePending might not be the best name.

Reply:
but what's the basic idea?   like, don't render this if there is no profile?

Reply:
yeah, consider it pending if theres no profile